doc/classes/EditorInterface.xml

```diff
@@ -24,6 +24,16 @@
 				Edits the given [Resource].
 			</description>
 		</method>
+		<method name="edit_script">
+			<return type="void" />
+			<argument index="0" name="script" type="Script" />
+			<argument index="1" name="line" type="int" default="-1" />
+			<argument index="2" name="column" type="int" default="0" />
+			<argument index="3" name="grab_focus" type="bool" default="true" />
+			<description>
+				Edits the given [Script]. The line and column on which to open the script can also be specified.
```

Comment:
Might be worth specifying how this works with user-configured external editor. And maybe add a backreference to it in `edit_resource` to edit [Script] resources?

Reply:
That's a good point, I've added that information. I didn't want the description of `edit_resource` to make it sound like they always have to use `edit_script` for [Script], but I hope it doesn't make users get confused about which one they should use.